activerecord/lib/active_record/serialization.rb

```diff
@@ -9,7 +9,7 @@ module Serialization
     end
 
     def serializable_hash(options = nil)
-      options = options.try(:clone) || {}
+      options = options.try(:dup) || {}
```

Comment:
Can we just remove this line and change the above line to `def serializable_hash(**options)`?